### PR TITLE
Add requirements-test.in

### PR DESCRIPTION
**Pull request by mpitcel**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -135,3 +135,4 @@ dmypy.json
 
 # build artifacts
 ssh
+.requirements.venv
```
Comment (baristaTam) [inline .gitignore:138]:
```suggestion
.requirements.venv

```

File: Makefile [hunk 2]
```diff
@@ -36,14 +37,22 @@ ssh/id_rsa: ssh
...
 	
 clean:
-	rm -rf venv ssh
+	rm -rf venv ssh $(VENV_REQS)
 	rm -f soar_null_router.tgz .tag
 	-find src -type d -name __pycache__ -exec rm -fr "{}" \;
 	git checkout -- $(TAG_FILES)
+
+force-clean: clean
+	rm -f requirements-test.txt
```
Comment (baristaTam) [inline Makefile:58]:
```suggestion
	rm -f requirements-test.txt
	
```

**Review by ddriddle — DISMISSED** [review]
Please remove vcrpy.

File: .gitignore (hunk 1 above)
Comment (baristaTam) [reply .gitignore:138]: missing eol

**Review by baristaTam — CHANGES_REQUESTED** [review]

**Review by gshere — DISMISSED** [review]

**Review by spaldr — APPROVED** [review]

**Review by zdc217 — APPROVED** [review]

**Pull request merged**